stable/postgresql/templates/statefulset-slaves.yaml

```diff
@@ -83,12 +83,19 @@ spec:
               chmod 700 {{ .Values.persistence.mountPath }}/data
               find {{ .Values.persistence.mountPath }} -mindepth 0 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | \
                 xargs chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }}
+{{- if .Values.shmVolume.enable }}
```

Comment:
What indentation would you like there?

Reply:
I would align it with the `chmod` command